Soccar [Client]/src/nl/soccar/ui/fx/GameCanvasFx.java

```diff
@@ -52,18 +65,47 @@ public void stop() {
         gameTimer.stop();
     }
 
+    /**
+     * Updates each WorldObject by calling their step method.
+     */
     private void update() {
-        List<Drawable> drawables = super.getDrawables();
-        drawables.stream().forEach(d -> d.update());
+
+        List<WorldObject> worldObjects = super.getWorldObjects();
+        GameStatus status = getGame().getStatus();
+
+        if (status == GameStatus.RUNNING) {
+            worldObjects.forEach(WorldObject::step);
+        }
+
+        if (status == GameStatus.SCORED) {

```

Comment:
Beschrijven waarom dit kan (het is raar als je kijkt naar de vorige if-statement).